### PR TITLE
405 - Create survey item and refactor survey list

### LINKED ISSUE #405: Create component Survey item and add to survey list

### PR DESCRIPTION
**Pull request by iurynogueira**
fix #405

# Create survey item and refactor survey list

Improve in card layout to surveys and added tests to the new survey list.

@edimossilva pls check my interfaces

### CODE REVIEW

File: frontend/src/api/models/survey.ts [hunk 1]
```diff
@@ -3,7 +3,15 @@ export interface QuestionType {
...
 
+export interface AnswersSurvey {
+  id: number;
+  status: string;
+  user_id: number;
+}
+
 export interface Survey {
+  answers_surveys: AnswersSurvey[] | null;
+  current_answers_survey: AnswersSurvey[] | null;
```
Comment (edimossilva) [inline frontend/src/api/models/survey.ts:14]: I guess this shouldnt be an array

File: frontend/src/components/main/surveys-list.test.tsx [hunk 2]
```diff
@@ -18,21 +18,24 @@ describe('Surveys list', () => {
       id: 1,
       name: 'Animals survey',
       description: 'Nice animals',
-      questions: [{
-        id: 1,
-        name: 'What is your favorite animal?',
-        question_type: {
+      answers_surveys: [],
+      current_answers_survey: [],
```
Comment (edimossilva) [inline frontend/src/components/main/surveys-list.test.tsx:22]: Confirmed, this should not be an array

**Comment by edimossilva** [comment]
I would suggest to rename the pr from "405 - survey item" to "405 - Create survey item and refactor survey list"

Using the title that you put on the description

**Comment by iurynogueira** [comment]
Thanks @edimossilva i fixed all sugestions.

**Pull request merged**